src/stdlib_sorting_ord_sort.fypp

```diff
@@ -0,0 +1,1419 @@
+#! Integer kinds to be considered during templating
+#:set INT_KINDS = ["int8", "int16", "int32", "int64"]
+
+#! Real kinds to be considered during templating
+#:set REAL_KINDS = ["sp", "dp", "qp"]
```

Comment:
```suggestion
#:include "common.fypp"
```
These variables are already defined in `common.fypp`

Reply:
Done!